armi/operators/tests/test_operators.py

```diff
@@ -103,37 +111,96 @@ def test_loadStateError(self):
         with self.assertRaises(RuntimeError):
             self.o.loadState(0, 1)
 
+    def test_setStateToDefault(self):
+
+        # reset the runType for testing
+        self.assertEqual(self.o.cs[CONF_RUN_TYPE], "Standard")
+        self.o.cs = self.o.cs.modified(newSettings={"runType": "fake"})
+        self.assertEqual(self.o.cs[CONF_RUN_TYPE], "fake")
+
+        # validate the method works
+        cs = self.o.setStateToDefault(self.o.cs)
+        self.assertEqual(cs[CONF_RUN_TYPE], "Standard")
+
+    def test_snapshotRequest(self):
+        with TemporaryDirectoryChanger():
+            self.o.snapshotRequest(0, 1)
+
+
+class TestTightCoupling(unittest.TestCase):
+    def setUp(self):
+        self.cs = settings.getMasterCs()
+        self.cs[CONF_TIGHT_COUPLING] = True
+        self.o = Operator(self.cs)
+        self.o.r = Reactor("empty", None)
+        self.o.r.core = Core("empty")
+
     def test_couplingIsActive(self):
-        """ensure that cs["tightCoupling"] controls couplingIsActive"""
-        self.assertFalse(self.o.couplingIsActive())
-        self.o.cs["tightCoupling"] = True
+        """ensure that cs[CONF_TIGHT_COUPLING] controls couplingIsActive"""
         self.assertTrue(self.o.couplingIsActive())
+        self.o.cs[CONF_TIGHT_COUPLING] = False
```

Comment:
above you have `self.cs` defined than an operator init-ed from that, so can this be `self.cs[CONF_TIGHT_COUPLING] = False`?

Reply:
It could since both `self.cs` and `self.o.cs` point to the same cs object in memory. My thoughts on the `self.o.cs` route: once we get rid of `set/getMasterCs`, `self.cs` and `self.o.cs` could be different. So the assertions should live on `self.o.cs` I think. What do you think?